src/modules/job-manager/jobtap.c

```diff
@@ -67,6 +67,26 @@ struct dependency {
 static int job_emit_pending_dependencies (struct jobtap *jobtap,
                                           struct job *job);
 
+
+static int errprintf (jobtap_error_t *errp, const char *fmt, ...)
```

Comment:
Nice cleanup.  It might be a good idea to put one of these in libutil along with a generic `struct error` for internal reuse, but it's fine if we save that for later.  Maybe we can can come up with something that helps us preserve errno too.

Anyway, this looks good.  Don't change.